src/ray/core_worker/core_worker.cc

```diff
@@ -380,8 +379,20 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
       }));
 
 #if defined(__APPLE__) || defined(__linux__)
-  experimental_mutable_object_manager_ =
-      std::make_shared<experimental::MutableObjectManager>();
+  // TODO(jhumphri): Combine with implementation in NodeManager.
+  auto raylet_channel_client_factory = [this](const NodeID &node_id) {
```

Comment:
Can you add a TODO here to pool these connections? Since we already create clients to other raylet elsewhere in CoreWorker.